src/runtime_src/driver/xclng/drm/xocl/devices.h

```diff
@@ -904,12 +904,14 @@ enum subdev_id {
 	{ XOCL_PCI_DEVID(0x10EE, 0x5000, PCI_ANY_ID, MGMT_XBB_DSA52) },	\
 	{ XOCL_PCI_DEVID(0x10EE, 0x5004, PCI_ANY_ID, MGMT_XBB_DSA52) },	\
 	{ XOCL_PCI_DEVID(0x10EE, 0x5008, PCI_ANY_ID, MGMT_XBB_DSA52) },	\
-	{ XOCL_PCI_DEVID(0x13FE, 0x006C, PCI_ANY_ID, MGMT_6A8F) },	\
+	{ XOCL_PCI_DEVID(0x13FE, 0x006C, PCI_ANY_ID, MGMT_XBB_DSA52) },	\
```

Comment:
Changing MGMT_6A8F to MGMT_XBB_DSA52 may introduce compatible issue with old DSA (13fe 6a). Could you confirm that you have tested your change with old DSA 13fe,6a?